capellambse/model/layers/oa.py

```diff
@@ -48,7 +48,7 @@ def outputs(self) -> c.ElementList[fa.FunctionalExchange]:
         return self._model.oa.all_activity_exchanges.by_source(self)
 
     @property
-    def exchanges(self) -> c.ElementList[fa.FunctionalExchange]:
+    def related_exchanges(self) -> c.ElementList[fa.FunctionalExchange]:
```

Comment:
That looks like an oversight from 59fc83257efcf78926f95234e6adebba81dc820c. IMO it should therefore be declared as "fix" rather than "refactor". Can you also add the "owned exchanges" as the new `exchanges` attribute? Should be a simple `DirectProxyAccessor` if I'm not entirely wrong.